src/tareas.ts

```diff
@@ -0,0 +1,76 @@
+interface Tasks {
+  id?: string;
+  title: string;
+  description: string;
+  complete?: boolean;
+}
+
+class List {
+  tasks: Tasks[] = [];
+
+  addTask(id: string, title: string, description: string): void {
+    const newTask: Tasks = {
+      id,
+      title,
+      description,
+      complete: false,
+    };
+
+    this.tasks.push(newTask);
+    `Nueva tarea agragada: ${title}`;
+  }
+
+  edit(id: string, task: Tasks, newTitle: string, newDescription: string) {
+    var editTask: void[] = this.tasks.map((ed, i) => {
+      if (ed.id === id) {
+        this.tasks[i] = {
+          ...task,
+          id,
+          title: newTitle,
+          description: newDescription,
+        };
+      }
+    });
+  }
+
+  delete(id: string) {
```

Comment:
?? Hay que ser consistente con los nombres que elijas

```suggestion
  deleteTask(id: string) {
```